connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -272,6 +274,14 @@ class SparkSession private[sql] (
    */
   def read: DataFrameReader = new DataFrameReader(this)
 
+  /**
+   * Interface through which the user may create, drop, alter or query underlying databases,
+   * tables, functions etc.
+   *
+   * @since 3.4.0
```

Comment:
1. still `@since 3.4.0` or `@since 3.5.0`

Reply:
Let's make it 3.5

Reply:
ok